templates/copytrading/Inspector.tsx

```
@@ -0,0 +1,117 @@
+import { useEffect, useState } from 'react'
+import { useRef } from 'react'
+import toast from 'react-hot-toast'
+import { MAX_TOKENS_LIMIT, SUPPORTED_NETWORKS } from './constants'
+import type { Config } from './types'
+
+export default function Inspector() {
+    const [config, updateConfig] = useFrameConfig<Config>()
+    const [tooManyTokens, setTooManyTokens] = useState(false)
+    const fid = useFarcasterId()
+
+    useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: config.walletAddress

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)